hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -54,13 +54,23 @@
 @RunWith(HazelcastParallelClassRunner.class)
 @Category(QuickTest.class)
 public class ClientMapBasicTest {
+    private static final String KEY = "key";
```

Comment:
I think this makes the whole test much easier to read, because you don't have any variations in the test files like "key", "Key", "KEY" etc. We totally don't care about the content in these cases. In my private projects I would even name those constants ANY_KEY, ANY_NEW_VALUE etc. to express, that it's not important what the content of the variable is. It is the usage/meta info which is the important thing. I can't find any better than a constant to express this (and it's not my idea, I got this from a book about testing).